source/reference/headings.txt

```diff
@@ -0,0 +1,155 @@
+.. _style-guide-headings:
+
+========
+Headings
+========
+
+.. default-domain:: mongodb
+
+Use over and underlines consisting of a repeating character to specify a heading
+in rST.
+
+Heading Level Markup
+--------------------
+
+The character used for the over and underlines has no effect on the heading
+level, as long as the order used is consistent. However, to maintain consistency
```

Comment:
Nitpick: "as long as the order used is consistent" is ambiguous. As someone who uses rST, I know what you're trying to say... but I'm not sure if interns and new writers would grok your point.

If everyone adheres to the ordering rules, it doesn't really matter. But we all know our docs aren't perfect, and if someone doesn't grok this point fully, they could end up very confused by page rendering vs. rst source.

Consider adding an example to demonstrate this idea.